src/Tasks/NativeMethods.cs

```diff
@@ -1125,7 +1125,7 @@ internal static bool AllDrivesMapped()
         /// <param name="dwLength">The size, in bytes, of the returned szBuffer.</param>
         /// <returns>HResult</returns>
         [DllImport(MscoreeDLL, SetLastError = true, CharSet = CharSet.Unicode)]
-        internal static extern uint GetFileVersion(String szFullPath, StringBuilder szBuffer, int cchBuffer, out uint dwLength);
+        internal static extern uint GetFileVersion(string szFullPath, char[] szBuffer, int cchBuffer, out uint dwLength);
```

Comment:
Again, this doesn't have `[out]`, so I think `StringBuilder` might be valid.

Reply:
https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/hosting/getfileversion-function

Reply:
@ladipro Could you point me in the right direction on this?